packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -37,6 +37,8 @@ export interface IMenuItemClassNames {
   linkContentMenu: string;
 }
 
+const CONTEXTUAL_SPLIT_MENU_WIDTH = '28px';
```

Comment:
Nit, this should be CONTEXTUAL_SPLIT_MENU_MINWIDTH

Reply:
> @jgage176 Please do npm run change and commit the change file - looks like that's what the build failed on.

Ah, it said to run that in the instructions but I didn't know what to do with the file - I didn't realize we needed to commit it - I tried to attach it to the PR but it didn't allow me to ;)